### PR TITLE
Modifications to RT1.26 basic_static_route_support_test.go

### PR DESCRIPTION
**Pull request by divyarema**
1 Updated deviations
2 Modified route next hop telemetry check on configuring static route next-hop recursive lookup to false.
3. Added udp to traffic flow for better ecmp load balancing

### CODE REVIEW

File: feature/staticroute/otg_tests/basic_static_route_support_test/basic_static_route_support_test.go [hunk 1]
```diff
@@ -343,14 +343,21 @@ func (td *testData) testRecursiveNextHopResolutionDisabled(t *testing.T) {
...
 	t.Run("Telemetry", func(t *testing.T) {
 
-		gnmi.Await(t, td.dut, sp.Static(td.staticIPv4.cidr(t)).Prefix().State(), 30*time.Second, td.staticIPv4.cidr(t))
-		gnmi.Await(t, td.dut, sp.Static(td.staticIPv6.cidr(t)).Prefix().State(), 30*time.Second, td.staticIPv6.cidr(t))
 		// Validate static route next-hop recursive lookup is disabled
-		if got, want := gnmi.Get(t, td.dut, sp.Static(td.staticIPv4.cidr(t)).NextHop("0").Recurse().State()), false; got != want {
-			t.Errorf("IPv4 Static Route next hop: got: %v, want: %v", got, want)
+		_, v4RecurseSet := gnmi.Watch(t, td.dut, sp.Static(td.staticIPv4.cidr(t)).State(), 30*time.Second, func(v *ygnmi.Value[*oc.NetworkInstance_Protocol_Static]) bool {
+			val, present := v.Val()
+			return !present || (present && !val.GetNextHop("0").GetRecurse())
```
Comment (LimeHat) [inline feature/staticroute/otg_tests/basic_static_route_support_test/basic_static_route_support_test.go:349]: can you elaborate on the purpose of this change?

File: feature/staticroute/otg_tests/basic_static_route_support_test/basic_static_route_support_test.go [hunk 2]
```diff
@@ -343,14 +343,21 @@ func (td *testData) testRecursiveNextHopResolutionDisabled(t *testing.T) {
...
+		}).Await(t)
+		if !v4RecurseSet {
+			t.Log("Unable to set recurse to false for v4 prefix")
```
Comment (LimeHat) [inline feature/staticroute/otg_tests/basic_static_route_support_test/basic_static_route_support_test.go:352]: seems like this should be `t.Errrorf`? if the recurse is not working as expected, why is it downgraded to a log message?

Comment (divyarema) [reply feature/staticroute/otg_tests/basic_static_route_support_test/basic_static_route_support_test.go]: There are more changes to file. This is modified to error in latest file.

File: feature/staticroute/otg_tests/basic_static_route_support_test/basic_static_route_support_test.go (hunk 1 above)
Comment (divyarema) [reply feature/staticroute/otg_tests/basic_static_route_support_test/basic_static_route_support_test.go:361]: Recurse is set to 'false' here. So next hop won't be resolved, and route won't be present. Code modified to 'handle this.

**Review by Swetha-haridasula — APPROVED** [review]
LGTM

**Pull request merged**